.github/workflows/go.yml

```diff
@@ -71,9 +73,9 @@ jobs:
       - name: 'GCAssert'
         # Only run gcassert on the latest versions of Go. Inlining heuristics
         # change from version to version.
```

Comment:
This comment seems out of date. We run gcassert on go1.18+.

Reply:
"the latest versions" was meant to imply that there might be a few recent versions on which we run gcassert, not just the single latest.